pkg/action/discoverer.go

```diff
@@ -190,6 +192,7 @@ func (d *Discoverer) getTargets(ctx context.Context) ([]*targetgroup.Group, erro
 						"project", project,
 						"vapp", vappName,
 						"server", server.Name,
+						"ID", server.ID,
```

Comment:
Same here

Reply:
applied in [8bb507d](https://github.com/promhippie/prometheus-vcd-sd/pull/31/commits/8bb507df716907a7279d67e92d7664cf918b5fd4)